src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -184,6 +184,8 @@ private static PackOptions GetPackOptions(PackageSpec packageSpec, JObject rawPa
             var rawPackOptions = rawPackageSpec.Value<JToken>(PackOptions) as JObject;
             if (rawPackOptions == null)
             {
+                packageSpec.Owners = new string[] {};
+                packageSpec.Tags = new string[] {};
```

Comment:
``new string[0]``?   It's probably the same result, it would just be more clear.